packages/dapp/src/hooks/useBlockchainExplorerURL.ts

```diff
@@ -5,13 +5,9 @@ export const useBlockchainExplorerURL = () => {
   const { account } = useWallet();
   const { getChainById } = useChains();
 
-  if (account.isActive && account.chainId) {
-    try {
-      return `${
-        getChainById(account.chainId).metamask.blockExplorerUrls[0]
```

Comment:
Do we know for which chainId it was throwing the error?

also, can we have a 
```ts
console.error(`No blockchain explorer found for ${account.chainId}`)
```

Reply:
If we don't have block explorer, please check [data-types](https://github.com/lifinance/data-types) package and add the missing one.

Reply:
@chybisov We do have a blockchain explorer available for each chain we provide - I have tested all of them. 
Apparently, this hook just fired before any chains were fetched - resulting in an error. But this seems to be a rare edge case but now, it should not throw any errors anymore because that if-clause is catching it.